src/content/developers/docs/programming-languages/rust/index.md

```diff
@@ -35,7 +35,7 @@ Need a more basic primer first? Check out [ethereum.org/learn](/learn/) or [ethe
 - [Build A Decentralized Chat Using JavaScript and Rust](https://medium.com/perlin-network/build-a-decentralized-chat-using-javascript-rust-webassembly-c775f8484b52)
 - [Build a Decentralized Todo App Using Vue.js & Rust](https://medium.com/@jjmace01/build-a-decentralized-todo-app-using-vue-js-rust-webassembly-5381a1895beb)
 
-- [An Intro to Secret Contracts](https://blog.enigma.co/getting-started-with-enigma-an-intro-to-secret-contracts-cdba4fe501c2)
+- [An Intro to Secret Contracts](https://docs.scrt.network/secret-network-documentation/development/secret-contract-fundamentals/secret-contracts-introduction)
```

Comment:
```suggestion
```

Thanks @aryankeluskar 

This article doesn't seem the most on topic given this is pretty specific to the cosmos ecosystem. While I agree with deprecating the old article that was here, I don't think this should replace it. Fine with just removing the article and not replacing it with anything for now until something more specific for Ethereum can be suggested here.